app/components/entryReview/EntryComments/Comment/index.tsx

```diff
@@ -62,12 +60,14 @@ function Comment(props: Props) {
             className={_cs(styles.commentContainer, className)}
             headingSize="extraSmall"
             headerActionsContainerClassName={styles.headerActions}
-            headerActions={mentionedUsersDetails.length > 0 && (
+            headerActions={mentionedUsers.length > 0 && (
                 <>
                     Assigned to
-                    <CommaSeparateItems
+                    <CommaSeparatedItems
                         className={styles.assignees}
-                        items={mentionedUsersDetails}
+                        items={mentionedUsers}
+                        keySelector={(mentionedUser) => mentionedUser.id}
+                        labelSelector={(mentionedUser) => mentionedUser.displayName ?? '?'}
```

Comment:
The anonymous function will always be created at every render and can cause serious performance penalties in general. Let's create these functions in the module scope or in the function scope.